src/nas_vrf_utils.cpp

```diff
@@ -47,7 +47,7 @@ static auto vrf_map = new std::unordered_map<const char *,nas_vrf_ctrl_t *, _vrf
 
 static std_rw_lock_t rw_lock = PTHREAD_RWLOCK_INITIALIZER;
 
-static inline void print_record(nas_vrf_ctrl_t*p) {
+static inline void print_record(const nas_vrf_ctrl_t *p) {
```

Comment:
Why u had to make it constant?

Reply:
Because print_record doesn't change anything, only prints. this is there to prevent accidental changing of *p and to improve clarity.